srcpkgs/python3-gnupg/template

```diff
@@ -1,18 +1,20 @@
 # Template file for 'python3-gnupg'
 pkgname=python3-gnupg
-_pkgname=python-gnupg
-version=0.4.6
-revision=4
-wrksrc=$_pkgname-$version
-build_style=python3-module
-hostmakedepends="python3-setuptools"
+version=0.4.9
+revision=1
+wrksrc=${pkgname/3/}-$version
+build_style=python3-pep517
+make_install_target=python_gnupg-${version}-py2.py3-none-any.whl
+hostmakedepends="python3-setuptools python3-wheel"
 depends="python3 gnupg"
+checkdepends="${depends}"
```

Comment:
Do you think using pytest could fix this?
```
=> python3-gnupg-0.4.9_1: running do_check ...
=> WARNING: Unable to determine tests for PEP517 Python templates=> python3-gnupg-0.4.9_1: running pre-install hook: 00-libdir ...
```

Reply:
Good catch! I didn't see that it wasn't running the tests after switching to pep517 for the 0.4.9 update.

Reply:
I'm a bit lost here, with

```suggestion
checkdepends="${depends} python3-pytest"
```
I now get this:

```text
=> python3-gnupg-0.4.9_1: running do_check ...
/void-packages/common/build-style/python3-pep517.sh: line 21: NO_EXTERNAL_TESTS=1: command not found
=> ERROR: python3-gnupg-0.4.9_1: do_check: '${make_check_pre} python3 -m pytest ${testjobs} ${make_check_args} ${make_check_target}' exited with 127
=> ERROR:   in do_check() at common/build-style/python3-pep517.sh:21
```

Reply:
resolved! pytest is now used, and with `env` in front of the variable, it works correctly.